src/test/mixxxtest.cpp

```diff
@@ -20,15 +20,10 @@ QString makeTestConfigFile(const QString& path) {
 QScopedPointer<MixxxApplication> MixxxTest::s_pApplication;
 
 MixxxTest::ApplicationScope::ApplicationScope(int& argc, char** argv) {
-    DEBUG_ASSERT(!s_pApplication);
-
-    s_pApplication.reset(new MixxxApplication(argc, argv));
-
-    SoundSourceProxy::registerSoundSourceProviders();
-
     // Construct a list of strings based on the command line arguments
     CmdlineArgs args;
-    DEBUG_ASSERT(args.Parse(argc, argv));
+    VERIFY_OR_DEBUG_ASSERT(args.Parse(argc, argv)) {
+    }
```

Comment:
Committed by mistake?

Reply:
No, the result was previously not verified.

VERIFY_OR_DEBUG_ASSERT always executes the expression, which is mandatory and intended here. I will rephrase it.

Reply:
Aaah, okay. I think I'd rather save the result in a local variable and debug assert that or just add a warning message to body :D